deploy-0chain/action.yml

```diff
@@ -447,6 +447,28 @@ runs:
           echo "==========================================================="
         fi
 
+    - name: "Install Zs3Server"
+      shell: 'script --return --quiet --command "bash {0}"'
+      run: |
+        ./zwallet faucet --methodName "pour" --tokens 9 --input "{}" --silent
+        ./zbox newallocation --lock 5 --data 2 --parity 2 --silent | grep -o "[a-z0-9]\{64\}" > allocation.txt
+        echo "Allocation hash: $(cat allocation.txt)"
+        echo "ALLOCATION=$(cat allocation.txt)" >> $GITHUB_ENV
+        echo "Allocation hash env variable: ${{ env.ALLOCATION }}"
+
+    - name: "Install Zs3Server"
+      shell: 'script --return --quiet --command "bash {0}"'
+      run: |
+        helm upgrade --install --wait \
+          --set zs3server.hostName=dev-${{ env.RUNNER_NUMBER }} \
+          --set zs3server.host=devnet-0chain.net \
+          --set zs3server.client.image.tag=v0.0.2 \
+          --set zs3server.logsearchapi.image.tag=v0.0.1 \
+          --set zs3server.minio.image.tag=v0.0.1 \
+          --set zs3server.minio.config.zcn_config.block_worker=https://${{ env.NETWORK_URL }}/dns \
+          --set zs3server.minio.config.allocation_1=${{ env.ALLOCATION }} \
```

Comment:
what is this for out of interest?

Reply:
It's a migration tool allowing clients to migrate data from S3 to an allocation